system/View/View.php

```diff
@@ -336,7 +336,7 @@ public function excerpt(string $string, int $length = 20): string
      */
     public function setData(array $data = [], ?string $context = null): RendererInterface
     {
-        if ($context) {
+        if ($context !== null && $context !== '') {
```

Comment:
cast to string

Reply:
Only `$context !== null` is sufficient.
`@phpstan-param null|'html'|'js'|'css'|'url'|'attr'|'raw' $context`

Reply:
updated to ony check !==null